app/src/main/java/com/compscieddy/meetinthemiddle/HomeActivity.java

```diff
@@ -60,6 +68,12 @@
 
   @Bind(R.id.group_recycler_view) RecyclerView mGroupRecyclerView;
   @Bind(R.id.map) MapView mMapView;
+  @Bind(R.id.app_bar_layout) AppBarLayout mAppBarLayout;
+  @Bind(R.id.collapsing_toolbar) CollapsingToolbarLayout mCollapsingToolbarLayout;
+  @Bind(R.id.map_cardView)
+  CardView mMapCardView;
```

Comment:
It's annoying that Studio automatically adds a newline here but please try to keep it all on one line